source/develop/infra/testing/ovirttestday3.1.md

```diff
@@ -160,7 +160,7 @@ Please check [Node_Release_Notes](/develop/projects/node/release-notes/) prior t
 
 ## oVirt Information Details
 
-Beta RPMs for Fedora 17 are available from <http://www.ovirt.org/releases/beta/fedora/17/>. In order to use it create a `/etc/yum.repos.d/ovirt-engine-beta.repo` file with the following content:
```

Comment:
Unfortunately this syntax variant does not produce clickable links with a relative URL and this is displayed literally as text.

Reply:
well, this was a removal :-) but yes, I'll look into them.

Reply:
It's not a removal, this line is replaced. But yes maybe that would have been clearer if I had added the comment on the + line,